tests/PySys/apt_plugin/apt_sequence_install_with_version/run.py

```diff
@@ -40,7 +40,7 @@ def setup(self):
         output = subprocess.check_output(["/usr/bin/apt-cache", "madison", "rolldice"])
 
         # Lets assume it is the package in the first line of the output
-        self.version = output.split()[2]  # E.g. "1.16-1+b3"
+        self.version = output.split()[2].decode("utf8") # E.g. "1.16-1+b3"
```

Comment:
Why was this breaking the test?

Reply:
I dint understand the issue either

Reply:
Yes, it was breaking because the output returned is in bytes.

```E: Version 'b'1.16-1build1'' for 'rolldice' was not found```,  the decode removes the extra `b`.